test/RSocketClientServerTest.cpp

```diff
@@ -87,7 +87,7 @@ TEST(RSocketClientServer, ServerGetsGarbage) {
   auto server = makeServer(std::make_shared<HelloStreamRequestHandler>());
   folly::SocketAddress address{"::1", *server->listeningPort()};
 
-  folly::ScopedEventBaseThread worker{"rsocket-client"};
+  folly::ScopedEventBaseThread worker;
```

Comment:
This was crashing the build until I removed it?  Was this causing anyone else issues?